### ISSUE TITLE
visulization function import_from_file has no use

### ISSUE THREAD

**Issue by MerktSimon** [PEtab-dev/PEtab#416]
The function `import_from_file` is currently not used at all. cf
https://github.com/PEtab-dev/PEtab/blob/40ca2a012295a8009ed72cca22b60048cacf4e7b/petab/visualize/helper_functions.py#L29

Imo it does not provide extra value because all the importing is done in plot_data_and_simulation
Since it has to be kept up to date if e.g. changes to functions called by this function happen, I was wondering if we might just delete it

**Comment by plakrisenko** [comment]
I agree. The only thing, maybe it could be valuable to extract lines 94 - 136 in a separate function for readability/tidiness (kind of instead of import_from_files)?
